pennylane/measurements/classical_shadow.py

```diff
@@ -195,10 +195,10 @@ def circuit():
             dev = qml.device("default.qubit", wires=2, shots=5)
 
             measurements1 = [qml.classical_shadow(wires=(0,1), seed=10)]
-            tape1 = qml.tape.QuantumTape(ops, measurements1)
+            tape1 = qml.tape.QuantumTape(ops, measurements1, shots=5)
```

Comment:
should `shots=5` be removed from the device above then?

Reply:
I remember I tried it once a while back and it caused an issue with some corner case. I figure it doesn't hurt to just leave it, plus users expect to have to give shots to the device.